pycbc/inference/entropy.py

```diff
@@ -6,9 +6,33 @@
 
 from scipy import stats
 
+def entropy(pdf1, pdf2=None, base=numpy.e):
```

Comment:
Yeah, I was mainly trying to avoid calling the same scipy function twice, because CodeClimate complains if two functions look very similar... I will find a way and change it.  
I have put the label "work in progress" because I will also make it an option to either give a number of bins and get the pdf with the current method or choose to do it via the KDE method.

Reply:
Sounds good. I've written up a KDE version myself if you want to compare, but it shouldn't be hard to do. Can we turn off code-climate for this?

What if you have entropy have options pdf1, base=numpy.e; and it calls KL but defaults to None.